src/platform/site-wide/cta-widget/tests/helpers.unit.spec.js

```diff
@@ -1,47 +1,39 @@
 import { expect } from 'chai';
 
-import localStorage from 'platform/utilities/storage/localStorage';
 import { eauthEnvironmentPrefixes } from 'platform/utilities/sso/constants';
 import { toolUrl, widgetTypes } from '../helpers';
 import environment from '../../../utilities/environment';
 
 describe('CTA helpers', () => {
   describe('A signed-in SSO user', () => {
-    beforeEach(() => {
-      localStorage.setItem('hasSessionSSO', true);
-    });
-    afterEach(() => {
-      localStorage.removeItem('hasSessionSSO');
-    });
-
     it('Download my data', () => {
-      expect(toolUrl(widgetTypes.HEALTH_RECORDS).url).to.equal(
+      expect(toolUrl(widgetTypes.HEALTH_RECORDS, true).url).to.equal(
```

Comment:
I would maybe declare a `const useSSOe = true` at the top of the file and pass that value so it's clear from the tests what's `true`